src/pudl/settings.py

```diff
@@ -650,72 +650,99 @@ class EtlSettings(BaseSettings):
 
     @root_validator(pre=False)
     def raw_table_validation_ferc1(cls, field_values):
-        """Require the presence of raw FERC1 tables needed for PUDL table creation."""
+        """Require the presence of raw FERC1 tables needed for PUDL table creation.
+
+        The FERC-derivative PUDL tables being loaded within this settings object
+        (within ``ferc_to_sqlite_settings``) are dependent on the presence of the tables
+        in the raw FERC SQLite databases, which are also a part of this settings object
+        (within ``datasets.ferc1``). A user doesn't always run ``ferc_to_sqlite`` and
+        ``pudl_etl`` at the same time, but this settings object has no idea what the
+        settings will be used for, so it validates that the tables that are in the
+        settings object are internally consistent.
+
+        If this settings object contains FERC-derivative PUDL tables AND there are
+        ``ferc_to_sqlite_settings``, this validator will run. If you are not attempting
+        to run ``ferc_to_sqlite``, you can either include no ``ferc_to_sqlite_settings``
+        or provide ``ferc_to_sqlite_settings`` that are consistent with your tables
+        in ``datasets.ferc1``.
+        """
         # only check if we are actually loading any pudl tables
-        if field_values["datasets"] and field_values["ferc_to_sqlite_settings"]:
-            ferc1_settings = field_values["datasets"].ferc1
-            ferc1_xbrl_to_sqlite_settings = field_values[
-                "ferc_to_sqlite_settings"
-            ].ferc1_xbrl_to_sqlite_settings
-            ferc1_dbf_to_sqlite_settings = field_values[
-                "ferc_to_sqlite_settings"
-            ].ferc1_dbf_to_sqlite_settings
-
-            pudl_etl_tables = ferc1_settings.tables
-            table_map = {
-                pudl_table: raw_dict
-                for (pudl_table, raw_dict) in pudl.TABLE_NAME_MAP_FERC1.items()
-                if pudl_table in pudl_etl_tables
-            }
+        if not field_values["datasets"] and not field_values["ferc_to_sqlite_settings"]:
```

Comment:
Why shouldn't this just check whether there's a `ferc1` dataset, rather than any dataset?

```py
if not field_values["datasets"].ferc1 and not field_values["ferc_to_sqlite_settings"]:
```

Reply:
bc the null version of `datasets` is `None` and this will not have a `ferc1` attribute.

but if datasets exist, `datasets.ferc1` can still be `None` so I think I need to add this in as well, but only after the upstream datasets check:

```
if (
    not field_values["datasets"]
    and not field_values["datasets"].ferc1
    and not field_values["ferc_to_sqlite_settings"]
):
    return field_values
```

Reply:
i settled on the latter bc it felt more succinct to me. I also changed the original `and` to an `or` bc either can be None for us to want to not do this validation